cla_backend/libs/eligibility_calculator/tests/test_calculator.py

```diff
@@ -1773,6 +1773,24 @@ def test_nass_benefit_is_not_eligible_and_category_isnt_immigration_and_disposab
         self.assertFalse(mocked_on_passported_benefits.called)
         self.assertTrue(mocked_on_nass_benefits.called)
 
+    def checker_with_assets(self, assets):
+        cd = fixtures.get_default_case_data()
+        cd['you'].update({'savings': dict(bank_balance=0, asset_balance=assets, investment_balance=0)})
+        case_data = CaseData(**cd)
+        return EligibilityChecker(case_data=case_data)
+
+    def test_cfe_request_with_no_assets(self):
```

Comment:
These CFE-specific tests seem a bit apart from the rest in class IsEligibleTestCase because they don't call is_eligible. How about putting them in a separate class DoCfeCivilCheckTestCase?